src/model/task/util.ts

```
@@ -5,8 +5,12 @@
 import { DEFAULT_PRIORITY_VALUE, DEFAULT_TYPE_VALUE, Task, TASK_WITH_DEFAULT_VALUES } from "@/model/task/schema";
 
 /**
- * @param parts - the task parts to merge.
- * @returns a new {@link Task} with the front-most non-default values taken from the parts.
+ * Merges multiple partial Task objects into a complete Task using a custom merge strategy.
+ *
+ * The function begins with default Task values and sequentially merges each provided partial Task. For each property, the front-most non-default value is retained. Special handling is applied for properties such as task type, priority, strings, DateTime objects, and Set collections.
```

ESLint

[error] Expected only 0 line after block description

(jsdoc/tag-lines)